README.md

```diff
@@ -27,60 +74,106 @@ mutation SignInMutation($input: signInInput!) {
 }
 ```
 
-Variables :
+It expects the following variables:
 
 ```json
-{ "input": { "email": "youremail", "password": "yourpassword" } }
+{
+  "input": {
+    "email": "your-email",
+    "password": "your-hashed-password"
+  }
+}
 ```
 
-The password needs to be hash client side using a salt. The salt needs to be retrieved with `GET /api/v1/users/myemail@mydomain.com`.
+**Example:**
 
-Once the salt is retrieved the hash password can be computed with bcrypt. In JavaScript:
-
-```javascript
-import bcrypt from 'bcryptjs';
+```bash
+$ curl 'https://api.sorare.com/graphql' \
+-H 'content-type: application/json' \
+-H 'x-csrf-token: <YourCSRFTokenValue>' \
+-H 'cookie: _sorare_session_id=<YourCookieValue>' \
+-d '{
+  "operationName": "SignInMutation",
+  "variables": { "input": { "email": "<YourEmail>", "password": "<YourHashPassword>" } },
+  "query": "mutation SignInMutation($input: signInInput!) { signIn(input: $input) { currentUser { slug email } errors { message } } }"
+}'
 
-bcrypt.hashSync(password, salt);
+{"data":{"signIn":{"currentUser":{"slug":"<YourSlug>","email":"<YourEmail>"},"errors":[]}}}
 ```
 
-You'll also need to pass a `_sorare_session_id` cookie and a `X-CSRF-TOKEN` token header. They can be retrieved with the same request as the salt.
+Should you keep the same `cookie` for future API calls, they will then be authenticated as this user.
 
-### With a cookie
+```bash
+$ curl 'https://api.sorare.com/graphql' \
+-H 'content-type: application/json' \
+-H 'x-csrf-token: <TheSameCSRFTokenValue>' \
+-H 'cookie: _sorare_session_id=<TheSameCookieValue>' \
+-d '{
+  "operationName": "CurrentUserQuery",
+  "query": "query CurrentUserQuery { currentUser { slug email } }"
+}'
 
-To use a cookie authentication you need to store the new `_sorare_session_id` and `X-CSRF-TOKEN` and pass it to all next requests.
+{"data":{"currentUser":{"slug":"<YourSlug>","email":"<YourEmail>"}}}
+```
+
+This is exactly the authentication mechanism that happens while browsing the [sorare.com](https://sorare.com) website.
+
+### GraphQL `signIn` mutation & JWT tokens
 
-### With a JWT token
+A better way than cookies to stay authenticated is to rely on [JWT tokens](https://jwt.io/).
```

Comment:
For short and long-lived authentication, you can request a [JWT](https://jwt.io) token.